### PR TITLE
osx-cpu-temp 1.1.0 (new formula)

### PR DESCRIPTION
**Pull request by stianeikeland**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by filler** [comment]
I was just about to add this myself.  Thanks for doing so!

File: Formula/osx-cpu-temp.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  url "https://github.com/lavoiesl/osx-cpu-temp/archive/1.1.0.tar.gz"
+  sha256 "94b90ce9a1c7a428855453408708a5557bfdb76fa45eef2b8ded4686a1558363"
+
+  def install
+    system "make"
+    bin.install "osx-cpu-temp"
+  end
+
+  test do
+    system "#{bin}/osx-cpu-temp"
```
Comment (javian) [inline Formula/osx-cpu-temp.rb]: Can we make some kind of check on the output of this command ?

Comment (equal-l2) [reply Formula/osx-cpu-temp.rb]: Something like this? (Provided the output is like `56.7°C`)
```
assert_match "°C", shell_output("#{bin}/osx-cpu-temp")
```

Comment (stianeikeland) [reply Formula/osx-cpu-temp.rb]: Sounds good, I'll add this. Maybe with a `-C`, to force it to celcius, just in case it follows locale.

**Comment by fxcoudert** [comment]
@BrewTestBot test this please

**Pull request closed without merging**